modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -193,6 +193,13 @@ ifdef::aws,azure,ash,gcp[]
 ----
 $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 ----
+ifndef::user-infra-vpc[]
++
+[IMPORTANT]
+====
+If you did not enable the Machine API for your cluster, you must remove the Kubernetes manifest files that define the worker machines. Otherwise, your cluster fails to install.
```

Comment:
suggest to update `Machine API` to `capability MachineAPI`

Reply:
Thanks, Jinyun. I see that capability exists in our [docs](https://docs.openshift.com/container-platform/4.14/installing/cluster-capabilities.html#machine-api-capability_cluster-capabilities). Thanks for pointing this out. I updated the clause accordingly. Please let me know your thoughts.

Reply:
LGTM